src/Command/ErrorFormatter/JunitErrorFormatter.php

```diff
@@ -65,9 +67,16 @@ public function formatErrors(
 	 *
 	 *
 	 */
-	private function createTestCase(string $reference, string $type, ?string $message = null): string
+	private function createTestCase(string $reference, string $type, ?string $message = null, ?string $fileName = null, ?int $lineNumber = null): string
 	{
-		$result = sprintf('<testcase name="%s">', $this->escape($reference));
+		$result = sprintf('<testcase name="%s"', $this->escape($reference));
+		if ($fileName !== null) {
+			$result .= sprintf(' file="%s"', $this->escape($fileName));
```

Comment:
What about absolute vs. relative path to the file? Typically you don't want to print the whole absolute path, but trim the beginning. But I'm not sure what PHPUnit does and what tools that read the output expect.

Anyway, this is usually done with the help of `RelativePathHelper`. Check the other formatters.